docs/development/testing_guide.rst

```diff
@@ -350,222 +502,218 @@ functions or pass in tuples containing inputs and expected values.
-when running ``import plasmapy`` from Python 2.7.
-
-To ignore a line of code in coverage tests, append it with
-``# coverage: ignore``.  If this comment is used on a line with a
-control flow structure (e.g., `if`, `for`, and `while`) that begins a
-block of code, then all lines in that block of code will be ignored.  In
-the following example, lines 3 and 4 will be ignored in coverage tests.
-
-.. code-block:: python
-  :linenos:
-  :emphasize-lines: 3,4
-
-  try:
-      import numpy
-  except ModuleNotFoundError as exc:  # coverage: ignore
-      raise RuntimeError from exc
-
-The ``.coveragerc`` file is used to specify lines of code and files that
-should always be ignored in coverage tests.
+Excluding lines in coverage reports
+-----------------------------------
 
-.. note::
+Occasionally there will be certain lines that should not be tested. For
+example, it would be impractical to create a new testing environment to
+check that an `ImportError` is raised when attempting to import a
+missing package. There are also situations that coverage tools are not
+yet able to handle correctly.
+
+To exclude a line from a coverage report, end it with
+``# coverage: ignore``. Alternatively, we may add a line to
+``exclude_lines`` in the ``[coverage:report]`` section of
+:file:`setup.cfg` that consists of a
+a pattern that indicates that a line be excluded from coverage reports.
+In general, untested lines of code should remain marked as untested to
+give future developers a better idea of where tests should be added in
+the future and where potential bugs may exist.
+
+Coverage configurations
+-----------------------
 
-  In general, untested lines of code should remain marked as untested to
-  give future developers a better idea of where tests should be added in
-  the future and where potential bugs may exist.
-
-Footnotes
-=========
-
-.. [1] In `Working Effectively With Legacy Code
-   <https://www.oreilly.com/library/view/working-effectively-with/0131177052/>`__,
-   Michael Feathers bluntly writes: "Code without tests is bad code.  It
-   doesn't matter how well written it is; it doesn't matter how pretty
-   or object-oriented or well-encapsulated it is.  With tests, we can
-   change the behavior of our code quickly and verifiably.  Without
-   them, we really don't know if our code is getting better or worse."
-
-.. [2] In the chapter "Bugs Are Missing Tests" in `Beyond
-   Legacy Code <https://pragprog.com/book/dblegacy/beyond-legacy-code>`__,
-   David Bernstein writes: "Every bug exists because of a missing test
-   in a system.  The way to fix bugs using TDD [test-driven development]
-   is first write a failing test that represents the bug and then fix
-   the bug and watch the failing test turn green.
+Configurations for coverage tests are given in the ``[coverage:run]``
+and ``[coverage:report]`` sections of :file:`setup.cfg`. Codecov_
+configurations are given in :file:`.codecov.yaml`.
+
+Best practices
+==============
```

Comment:
Similarly, is this section worth including here, or should I make this available in a different way outside of our documentation?